// 鉴权
module.exports = async (ctx, next) => {
    return next().catch((err) => {
        if (401 === err.status) {
            ctx.body = {
                code: 50001,
                message: '用户鉴权失败',
            };
        } else {
            throw err;
        }
    });
}
